gradle.properties

```diff
@@ -1,4 +1,4 @@
-release_version=2.6.4
+release_version=2.6.5
```

Comment:
I think we need to increase the minor version because the grpc-common version was updated

Reply:
I'm agree